scripts/install-build-deps

```diff
@@ -187,7 +187,12 @@ if ! $use_ansible ; then
    exit 0
 elif ! which ansible &>/dev/null ; then
     case $(distro_type) in
-        redhat) yum -y install epel-release
+        redhat)
+                source /etc/os-release
+                if [[ $NAME == "Rocky Linux" && $VERSION == "8.4 (Green Obsidian)" ]]; then
```

Comment:
@bdekvadiya : Suppose some one try it on RL 8.3, then too it shall work.
Note that this epel is not specific to Rocky linux.

Something similar to following can be used, if on first attempt epsl-release is not available.
yum-config-manager --add-repo https://archives.fedoraproject.org/pub/archive/epel/$VERSION/Everything/$ARCH/

Reply:
Hi @madhavemuri , I have updated the change as per your suggestion